cvat/apps/engine/media_extractors.py

```diff
@@ -264,7 +291,7 @@ def _get_av_container(self):
             self._source_path[0].seek(0) # required for re-reading
         return av.open(self._source_path[0])
 
-    def get_preview(self):
+    def get_preview(self, dimension=DimensionType.DIM_2D):
```

Comment:
If you have LidarDataReader, you will not need the extra argument. It isn't necessary here in VideoReader for sure.